psknode/bin/scripts/pskApiHubLauncher.js

```diff
@@ -2,7 +2,9 @@ const TAG = "API-HUB";
 
 let path = require("path");
 
-require("../../core/utils/pingpongFork").enableLifeLine();
+require("../../core/utils/pingpongFork").enableLifeLine({
```

Comment:
Prevent the lifeline from handling uncaught exceptions - this will be done by our script